enhancements/platform-validation.md

```diff
@@ -275,15 +254,15 @@ Three solutions mentioned in previous comments: Kind, k3d/k3s, microshift.
 
 * Kind is current available solution in Kelper. It works well on BM, but has known issues in VM.
```

Comment:
Which issues?

Reply:
Here is not so accurate description, actually the issue is for VM's power detection, let me remove such words later.